polaris-migrator/src/migrations/replace-sass-animatable/replace-sass-animatable.ts

```diff
@@ -244,17 +261,27 @@ export default createSassMigrator(
       }
 
       if (node.type === 'function') {
-        const easingFuncHandlers = {
-          [namespace('easing', options)]: migrateLegacySassEasingFunction,
-          // Per the spec, these can all be functions:
-          // https://w3c.github.io/csswg-drafts/css-easing/#easing-functions
-          linear: insertUnexpectedEasingFunctionComment,
-          'cubic-bezier': insertUnexpectedEasingFunctionComment,
-          steps: insertUnexpectedEasingFunctionComment,
-        };
+        if (node.value === namespace('easing', options)) {
+          migrateLegacySassEasingFunction(node, decl);
+          return;
+        }
 
-        if (isKeyOf(easingFuncHandlers, node.value)) {
-          easingFuncHandlers[node.value](node, decl);
+        if (ignoreUnknownFunctions) {
```

Comment:
Discovered a bug where `transition-timing-function: nonsense` was not flagged as an error. This flag (and surrounding logic catches it).